WordPress/gutenberg
# [RNMobile] - Blog layout template

`Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/2114

## Description
This PR adds the `Blog` layout since the `Latest Posts` block is now available.

## Screenshots

<details><summary>Blog Layout Preview</summary>

Blog preview Android | Blog preview Android
-|-
<img src="https://user-images.githubusercontent.com/4885740/78564450-bcca1880-781c-11ea-8d76-774dcdb1762b.png" width="250" /> | <img src="https://user-images.githubusercontent.com/4885740/78564454-bdfb4580-781c-11ea-8762-4a774b9f5770.png" width="250" />

Blog preview iOS | Blog preview iOS
-|-
<img src="https://user-images.githubusercontent.com/4885740/78564392-a6bc5800-781c-11ea-9484-8c1a349bc2ae.png" width="250" /> | <img src="https://user-images.githubusercontent.com/4885740/78564400-a91eb200-781c-11ea-84e3-4f48a4dba70f.png" width="250" />

</details>

<details><summary>Emoji compatibility check</summary>

### Android

API 21 | API 23
-|-
<img src="https://user-images.githubusercontent.com/4885740/78565082-ac666d80-781d-11ea-96df-5ee4a1877b09.png" width="250" /> |<img src="https://user-images.githubusercontent.com/4885740/78565087-af615e00-781d-11ea-99bb-fd35b1ea4451.png" width="250" />

API 27 | API 29
-|-
<img src="https://user-images.githubusercontent.com/4885740/78565160-c738e200-781d-11ea-9d1e-dc30a86659d3.png" width="250" /> |<img src="https://user-images.githubusercontent.com/4885740/78565164-ca33d280-781d-11ea-91d8-46a01a21b408.png" width="250" />

### iOS
<img src="https://user-images.githubusercontent.com/4885740/78565303-ffd8bb80-781d-11ea-945a-179b352ac6e8.png" width="250" /> 

</details>

## How has this been tested?
- Open the WordPress app with metro running
- Go to Pages
- Add a new page
- **Expect** to see the blog option in the layout picker, tap on it
- **Expect** to see the preview of the template
- Tap on apply
- **Expect** to see the layout applied in the editor
- Preview the page
- **Expect** to see the layout showing correctly

## Types of changes
New layout template

## Checklist:
